src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -6,14 +6,45 @@ import ErrorableFileInput from '../../common/components/form-elements/ErrorableF
 import ProgressBar from '../../common/components/ProgressBar';
 import { scrollAndFocus } from '../../common/utils/helpers';
 import { PhotoReviewDescription } from '../helpers.jsx';
+import _ from 'lodash/fp';
 
 const MIN_SIZE = 350;
 const SMALL_CROP_BOX_SIZE = 240;
 const LARGE_CROP_BOX_SIZE = 300;
 const WARN_RATIO = 1.3;
 const LARGE_SCREEN = 1201;
 
-function getCanvasDataForMove({ canvasData, cropBoxData, moveX, moveY }) {
+function getCanvasDataForDefaultPosition({ canvasData, cropBoxData, containerWidth }) {
```

Comment:
I need to set the default position in more than one place so I pulled the data creation into here.